project/AppDependencies.scala

```diff
@@ -22,7 +22,7 @@ object AppDependencies {
     "com.typesafe.play"      %% "play-test"              % current,
     "org.pegdown"            % "pegdown"                 % "1.6.0" ,
     "org.scalatestplus.play" %% "scalatestplus-play"     % "5.1.0",
-    "com.github.tomakehurst" % "wiremock-standalone"     % "2.27.2",
+    "com.github.tomakehurst" % "wiremock-standalone"     % "3.0.1",
```

Comment:
it's weird this one didn't complain without changing to `org.wiremock` - it may be worth changing just in case it happens in the future

Reply:
done